packages/logging/service.go

```diff
@@ -89,6 +128,71 @@ func (s *loggingService) GetForDogu(request *pb.DoguLogMessageRequest, server pb
 	return writeLogLinesToStream(s.logProvider, doguName, linesCount, server)
 }
 
+func (s *loggingService) SetLogLevel(ctx context.Context, req *pb.LogLevelRequest) (*emptypb.Empty, error) {
+	doguName := req.DoguName
+
+	if strings.TrimSpace(doguName) == "" {
+		return nil, createInternalErr(errors.New(responseMessageMissingDoguname), codes.InvalidArgument)
```

Comment:
I am not sure how the admin dogu handles the errors provided here. But I see no hint of the original task being done here (setting a log level of a specific dogu).

If several dogu log levels would be set really fast, would the administrator know for which dogu which error was returned? Putting me into the administrator's boots, I would have a certain interest to know which of the actions I need to inspect in more detail or retry.

There are a couple of ways to handle this like duplicating the error handling or extracting it into a wrapper method. But I think get the gist of it:

```golang
	errMsg := "could not apply log level for dogu %s and restart because an error occurred: "
...
	if err != nil {
		return nil, createInternalErr(fmt.Errorf(errorMsg + "unable to set log level: %w", err), codes.Internal)
	}
```